cmd/minishift/cmd/start_preflight.go

```diff
@@ -490,7 +496,6 @@ func checkIsoUrl() bool {
 		}
 	}
 
-	fmt.Printf("\n   Checking if %s exists ... ", strings.TrimPrefix(isoUrl, "file://"))
 	_, err := os.Stat(strings.TrimPrefix(isoUrl, "file://"))
```

Comment:
Here is a bug on Windows. You are enforcing a Windows path to be specified with forward slashes, aka file://c:/path/to/iso/my.iso. Only in this format, you let Windows paths get to the os.Stat call. There you trim off the file prefix and do an os.Stat with c:/path/to/iso/my.iso. I believe this is not going to work. Here is the result of the unit tests as run on AppVeyor:

    --- FAIL: TestIsoUrlOnWindows (0.00s)
	start_preflight_windows_test.go:48: Expected 'true' for given url 'file://C:/gopath/src/github.com/minishift/minishift/test/testdata/dummy.iso'. Got 'false'.

When dealing with file path, it is absolute essential to tests this across platforms. At least manually, preferably in a unit/integration test.

On a side node, we also have filehelper.Exists which wraps os.Stat in a function returning true or false.

Reply:
No, i did not test this in windows locally, that bug i see it now. I will integrate the PR and test it. Thanks @hferentschik